docs/content/extras/shortcodes.md

```diff
@@ -164,10 +166,12 @@ To create a shortcode, place a template in the layouts/shortcodes directory. The
 template name will be the name of the shortcode.
 
 In creating a shortcode, you can choose if the shortcode will use _positional
-parameters_ or _named parameters_ (but not both). A good rule of thumb is that if a
+parameters_ or _named parameters_ or _both_. A good rule of thumb is that if a
```

Comment:
Are you sure about this? I'm pretty sure I got test cases that verifies an error in these cases.